soroban-token-spec/src/lib.rs

```diff
@@ -2,6 +2,56 @@
 
 mod tests;
 
+use soroban_sdk::{contractevent, Address, BytesN, String};
+
+// Events that are a special case generated by the Stellar Asset Contract, but are part of SEP-41,
+// but are not expected to be generated by other contracts. Defined here privately only for the
+// purpose of generating the complete contract spec.
+
+/// Transfer event published when a classic payment uses the MEMO_TEXT memo type.
+#[contractevent(topics = ["transfer"], data_format = "map", export = false)]
+#[doc(hidden)]
+pub struct TransferWithMuxedString {
```

Comment:
Could these go into stellar-asset-spec instead to signify that they only concern Stellar assets?